### DISCUSSION TITLE
Can phone numbers be clickable?

### DISCUSSION THREAD

**Discussion by rafnov** [FossifyOrg/General-Discussion#458] [Ideas]
How to make phone number click point directly to any/default phone app?
Now I need to enter task/event, mark the number and then tap "call" via Android function.
Could it be just shorted to tap the number and open default phone app to call?

**Comment by Aga-C** [comment]
It's not possible right now.

**Comment by rafnov** [comment]
Why already closing :(
Can't it be converted to feature request? Should I post a new one to get more info?
Is the feature I'm about planned or not at all?

**Reply by Aga-C** [reply]
Doing it may be already a part of https://github.com/FossifyOrg/Calendar/issues/160 or https://github.com/FossifyOrg/Calendar/issues/12. If you see a possibility to do it separately, you can create a new feature request.
